<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="map_name" default="duckietown_map.pkl" />
    <arg name="draw_solution" default="False" />
	<param name="map_name" value="$(arg map_name)" />
	<param name="draw_solution" value="$(arg draw_solution)" />

	<node
                ns="$(arg veh)"
		name="graph_search_server"
		pkg="navigation"
		type="graph_search_server.py"
		output="screen"
		clear_params="true" required="true"
	/>

    <node
                ns="$(arg veh)"
		name="actions_dispatcher"
		pkg="navigation"
		type="actions_dispatcher.py"
		output="screen"
		clear_params="true" required="true"
	/>

</launch>
